Jenkinsfile

```diff
@@ -43,6 +43,14 @@ pipeline {
       }
     }
 
+    stage('Validate') {
```

Comment:
Looks like the `Validate` stage still needs to be moved up in the sequence, maybe even before `Build client Docker image`?